### PR TITLE
pl-PL work in progress

### PR DESCRIPTION
**Pull request by donpepe0**
pl-PL work in progress

### CODE REVIEW

File: data/language/pl-PL.txt [hunk 1]
```diff
@@ -1,179 +1,773 @@
 # STR_XXXX part is read and XXXX becomes the string id number.
 # Everything after the colon and before the new line will be saved as the string.
 # Use # at the beginning of a line to leave a comment.
+STR_0000    :
+STR_0001    :{STRINGID} {COMMA16}
+STR_0002    :Spiral Roller Coaster
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt]: If a string is missing, the game will fall back to English. It's therefore pointless to add untranslated strings. Please translate them.

Comment (donpepe0) [reply data/language/pl-PL.txt]: Sure, work in progress. This is rather for comparison purpose, as I can compare with other 100% translations line by line.

File: data/language/pl-PL.txt [hunk 2]
```diff
@@ -4056,37 +5230,9 @@ STR_SCNR    :Six Flags over Texas
 STR_PARK    :Six Flags over Texas
 STR_DTLS    :Wybuduj brakujące obiekty {OPENQUOTES}Six Flags{ENDQUOTES} lub upiększ lunapark własnymi projektami! Tylko nie zapomnij o ostatecznym celu - musisz przyciągnąć do parku więcej gości!
 
-<Build your own Six Flags Belgium>
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt]: Why did you remove these translations and replace them with English?

**Comment by Gymnasiast** [comment]
What are you plans with this pull request? I cannot merge it if it adds untranslated strings or removes translations - as I said, it falls back to English automatically. You should only add translated strings.

Do you intend to translate all strings, or only a few? If it's the latter, please remove the untranslated English strings.

(Please don't be put off by these comments. I'm happy that someone is finally working on Polish again. I'm saying these things to improve the translation.)

**Comment by donpepe0** [comment]
I'm planning to translate it all. I just updated pl-PL.txt so the string order match en-GB.txt. Then I can easily compare strings between the files. I thought that this may help other Polish folks too (if they are any). If there is switch back to English when string is missing, so why to bother keep only translated strings? Anyway you can discard PR, and I will do another one when finish. Thanks.

File: data/language/pl-PL.txt (hunk 2 above)
Comment (donpepe0) [reply data/language/pl-PL.txt]: A mistake, thanks I will fix it.

**Comment by Gymnasiast** [comment]
Nah, I'll just keep this PR open while you work on it.

**Comment by Gymnasiast** [comment]
Could you continue work on this?

**Comment by Zcooger** [comment]
I can continue if the progress stops. I've translated all strings for OpenLoco last night so I'll help there.

**Comment by Gymnasiast** [comment]
@Zcooger Oh, yes please! I just merged this one, please make sure you base your changed on the most recent version of `pl-PL`. Thanks in advance!
